Add `COCOTB_LOG_PREFIX` to customize the default log Formatter's appearance (#4900)
cocotb/cocotb: docs/source/building.rst

        If the tests pass, your simulator and version apply inertial writes as expected and you can turn on :envvar:`COCOTB_TRUST_INERTIAL_WRITES`.

.. envvar:: COCOTB_LOG_PREFIX

    Customize the log message prefix.

>     The value of this variable should be a Python f-string that has access to the following variables:

Reviewer: ```suggestion
    The value of this variable should be in Python f-string syntax. It has access to the following variables:
```
